packages/openapi-to-graphql/src/schema_builder.ts

```diff
@@ -889,6 +905,33 @@ export function getArgs({
     }
   }
 
+  // Add limit argument
+  if (
+    data.options.addLimitArgument &&
+    typeof operation.responseDefinition === 'object' &&
+    operation.responseDefinition.schema.type === 'array' &&
+    // Only add limit argument to lists of object types, not to lists of scalar types
+    ((operation.responseDefinition.subDefinitions as DataDefinition).schema
+      .type === 'object' ||
+      (operation.responseDefinition.subDefinitions as DataDefinition).schema
+        .type === 'array')
+  ) {
+    // Make sure slicing arguments will not overwrite preexisting arguments
+    if ('limit' in args) {
+      handleWarning({
+        typeKey: 'LIMIT_ARGUMENT_NAME_COLLISION',
+        culprit: `${operation.method.toLocaleUpperCase()} ${operation.path}`,
+        data,
+        log: translationLog
+      })
+    } else {
+      args['limit'] = {
+        type: GraphQLInt,
+        description: `Auto-generated argument that limits size of returned list of objects/lists`
```

Comment:
Here, I think we need to convey the semantics of limit (`Selects the first n elements from a list`) to the end-user!